hz.ps1

```diff
@@ -660,9 +705,16 @@ function ensureJar($jar, $repo, $artifact) {
     $script:classpath = $classpath
 }
 
-# say hello
-Write-Output "Hazelcast .NET Command Line"
-Write-Output ""
+# clear source file
+function clrSrc($file) {
```

Comment:
dead code?

Reply:
not really - from time to time I normalize the whole codebase - so I'd rather keep the method around and one day I'll update the hz script with some sort of "normalize" function